mllib/src/main/scala/org/apache/spark/mllib/optimization/NNLS.scala

```diff
@@ -53,8 +53,11 @@ private[spark] object NNLS {
    * projected gradient method.  That is, find x minimising ||Ax - b||_2 given A^T A and A^T b.
    *
    * We solve the problem
+   *
+   * {{{
    *   min_x      1/2 x^T ata x^T - x^T atb
-   *   subject to x &gt;= 0
+   *   subject to x >= 0
+   * }}}
```

Comment:
- Java
  <img width="653" alt="2016-11-27 3 58 59" src="https://cloud.githubusercontent.com/assets/6477701/20642584/f21ca3a6-b455-11e6-9575-81491f2abbdc.png">

Scaladoc not found